saylua/templates/layout.html

```diff
@@ -40,59 +41,62 @@
 </div>
 {% block container %}
 <div id="main-body" class="main-body">
-  <div id="notifications-container" class="notifications-container">
-    {% with messages = get_flashed_messages(with_categories=true) %}
-      {% if messages %}
-        {% for category, message in messages %}
-          {% if category == 'error' %}
-            <div class="notification error">Error: {{ message }} <a class="close-button" href="#" title="close">&#x00d7;</a></div>
-          {% else %}
-            <div class="notification">{{ message }} <a class="close-button" href="#" title="close">&#x00d7;</a></div>
-          {% endif %}
-        {% endfor %}
-      {% endif %}
-    {% endwith %}
-  </div>
-  {% if not fullscreen %}
-    {% include "sidebar.html" %}
-  {% endif %}
-  <div id="main-body-content" class="main-body-content">
-    {% block body %}{% endblock %}
+  {% include "sidebar.html" %}
+  <div id="main-body-column" class="main-body-column">
+    <div id="notifications-container" class="notifications-container">
+      {% with messages = get_flashed_messages(with_categories=true) %}
+        {% if messages %}
+          {% for category, message in messages %}
+            {% if category == 'error' %}
+              <div class="notification error">Error: {{ message }} <a class="close-button" href="#" title="close">&#x00d7;</a></div>
+            {% else %}
+              <div class="notification">{{ message }} <a class="close-button" href="#" title="close">&#x00d7;</a></div>
```

Comment:
These two should really just be condensed together into one notification with variable class names. That predates this PR though and is unrelated.